torch/csrc/jit/passes/onnx/helper.cpp

```diff
@@ -186,7 +186,13 @@ Node* transformToONNXConcatNode(
   for (auto* input : lc_node->inputs()) {
     auto new_input =
         need_new_input ? g->addInput()->copyMetadata(input) : input;
-
+    // Skip unsqueeze for input node if input dim is already 1 or more
```

Comment:
I don't understand why we need unsqueeze in the other case. Could you expand the comment on line 182 to explain?

Reply:
Expanded comment